.gitignore

```diff
@@ -16,4 +16,8 @@ gitlab/
 
 # ignore vscode files (debug config etc...)
 /.vscode
-/.idea
\ No newline at end of file
+/.idea
+
+# helm chart dependencies
+**/charts/*.tgz
+**/charts/**/requirements.lock
```

Comment:
```suggestion
cluster/charts/crossplane/charts/
cluster/charts/crossplane/requirements.lock
```

I think that would be easier to understand.

Reply:
We would theoretically want to guard against vendoring of a dependency of any chart that existed in this repo though right?

Reply:
> We would theoretically want to guard against vendoring of a dependency of any chart that existed in this repo though right?

Yes, I think so, too. They would all go into `cluster/charts/crossplane`. If we have a completely separate chart, then many changes will be due anyway.

Reply:
@muvaf oh I was saying using `**/` seemed more future proof :sweat_smile:

Reply:
Yeah, got it 😄  I was just saying that if there will be another directory than `cluster/charts/crossplane`, then many changes will be needed anyway, so it isn't worth the asterisk. But it's a very small detail anyway, you can merge it as is.